src/Infrastructure/Repositories/AttendeeRepository.cs

```diff
@@ -36,4 +38,50 @@ public Task<bool> AttendeeHasRoleAsync(int attendeeId, AttendeeRoleEnum role)
 			x.Id == attendeeId 
 		    && x.Roles.Any(r => r.RoleEnum == role));
 	}
+	
+	public async Task<bool> IsUserAttendeeOfConference(int userId, int conferenceId, CancellationToken cancellationToken)
+	{
+		return await dbContext.Attendees
+			.AnyAsync(a => a.UserId == userId && a.ConferenceId == conferenceId, cancellationToken);
+	}
+	
+	public async Task<PagedList<AttendeeInfoModel>> GetParticipantInfoModels(
+		int conferenceId, int page, int pageSize, CancellationToken cancellationToken)
+	{
+		var baseQuery = dbContext.Attendees
+			.Where(a => a.ConferenceId == conferenceId)
+			.OrderBy(a => a.Id);
+
+		var total = await baseQuery.CountAsync(cancellationToken);
+
+		var attendeeIds = await baseQuery
```

Comment:
po co 2 listy pobierasz?
```suggestion
		var atendees = await dbContext.Attendees
		.Where(a => a.ConferenceId == conferenceId)
		.OrderBy(a => a.Id)
		.Skip((page -1) * pageSize)
		.Take(pageSize)
		.Select(a => new AttendeeInfoModel
		{
			Id = a.Id,
			Name = a.UserSnapshot?.Name,
			Degree = a.UserSnapshot?.Degree,
			Country = a.User?.Country,
			Position = a.UserSnapshot?.Position,
			Workplace = a.UserSnapshot?.Workplace,
			IsAcademic = a.UserSnapshot?.IsPositionAcademic,
			Roles = a.Roles.Select(r => r.RoleEnum.ToString()).ToList(),
			PapersCount = a.Papers.Count(),
			ReviewsCount = a.Reviews.Count(),
			RegisteredAt = a.CreatedAt
		})
		.ToListAsync(cancellationToken)
```